front_end/src/components/layout/Navbar.js

```diff
@@ -52,6 +52,13 @@ class Navbar extends Component {
             </Button>
           </Link>
         </Grid>
+        <Grid item>
+          <Link to="/booking-not-logged-in">
```

Comment:
please change the way of displaying the button/link
I'd make it a link (similar to Deals of the Week), but show this link for all page sizes. (deal and feat. city is not displayed on smaller screens)